src/plugins/persistence/persistence.js

```diff
@@ -48,14 +37,13 @@ class Persistence extends Plugin {
 
 						const config = {
 							attachTo: angular.element(this.$document[0].body), // eslint-disable-line no-undef
-							controller: PersistencePanelController,
+							controller: () => {},
```

Comment:
can we just use persistenceView in controller and inside of controller make $scope.persistence = new PersView()?
so no need to write $persistence.view?